android/src/main/java/com/lwansbrough/RCTCamera/RCTCamera.java

```diff
@@ -131,6 +163,46 @@ public int getPreviewHeight(int type) {
         }
         return result;
     }
+    public Camera.Size getSmallestVideoSize(int type)
+    {
+        Camera camera = _cameras.get(type);
+        Camera.Size result = null;
+        if(camera == null) {
+            return null;
+        }
+        List<Camera.Size> sizes = getSupportedSizesWithFallback(camera);
+        if (null != sizes) {
+            for (Camera.Size size : sizes) {
+                if (result == null) {
+                    result = size;
+                } else {
+                    int resultArea = result.width * result.height;
+                    int newArea = size.width * size.height;
+
+                    if (newArea < resultArea) {
+                        result = size;
+                    }
+                }
+            }
+        }
+        return result;
+    }
+
+    public List<Camera.Size> getSupportedSizesWithFallback(Camera camera)
```

Comment:
This method is used for picture capturing also, but getSupportedPistureSizes is not used.